---
layout: home
title: Home
---

# About

I am a PhD candidate in the Department of Molecular Genetics at the
University of Toronto (co-supervised by [Ben
Blencowe](http://sites.utoronto.ca/intron) and [Quaid
Morris](http://www.morrislab.ca)). My research is focused on developing
computational and machine learning methods to study the transcriptome of the cell using high-throughput sequencing data.

Prior to starting my PhD, I was a research intern with the
Microsoft Biology Initiative group at
[Microsoft Research](https://research.microsoft.com) where I
developed [bioinformatics software](https://seqcos.codeplex.com/) using
.NET.

# Research Interests

- Computational biology
- Post-transcriptional regulatory mechanisms
- Machine learning

# Education

- **PhD**, Molecular Genetics, University of Toronto, 2011 - present
- **MSc**, Human Genetics and Bioinformatics, McGill University, 2008 - 2011
- **BSc**, Bioinformatics, University of Waterloo, 2003 - 2008

# Selected Publications

<small>\* denotes joint authorship. Full publication list available on
[Google Scholar](https://scholar.google.ca/citations?user=9xTTM_cAAAAJ).</small>

 1. **QAPA: a new method for the systematic analysis of alternative polyadenylation from RNA-seq data**  
   <small>**Ha KCH**, Blencowe BJ, Morris Q. Genome Biology (in press)</small>

 1. **An atlas of alternative splicing profiles and functional associations reveals new regulatory programs and genes that simultaneously express multiple major isoforms**  
   <small>Tapial J, **Ha KCH\***, Sterne-Weiler T\*, Gohr A\*, Braunschweig U,
Hermoso-Pulido A, Quesnel-Vallières M, Permanyer J, Sodaei R, Marquez
Y, Cozzuto L, Wang X, Gómez-Velázquez M, Rayon T, Manzanares M,
Ponomarenko J, Blencowe BJ, Irimia M. Genome Res. 2017;27:1759-1768.</small>

 1. **RNAcompete-S: Combined RNA sequence/structure preferences for RNA binding proteins derived from a single-step in vitro selection**  
    <small>Cook KB, Vembu S, **Ha KCH**, Zheng H, Laverty KU, Hughes TR, Morris Q. Methods. 2017;126:18–28.</small>

 1. **MECP2 Is Post-transcriptionally Regulated during Human Neurodevelopment by Combinatorial Action of RNA-Binding Proteins and miRNAs**  
    <small>Rodrigues DC, Kim D-S, Yang G, Zaslavsky K, **Ha KCH**, Mok RSF, Ross PJ, Zhao M, Piekna A, Wei W, Blencowe BJ, Morris Q, Ellis J. Cell Reports. 2016;17:720–34.</small>

1. **Identification of gene fusion transcripts by transcriptome sequencing in BRCA1-mutated breast cancers and cell lines**  
   <small>**Ha KCH**, Lalonde E, Li L, Cavallone L, Natrajan R, Lambros MB, Mitsopoulos C, Hakas J, Kozarewa I, Fenwick K, Lord CJ, Ashworth A, Vincent-Salomon A, Basik M, Reis-Filho JS, Majewski J, Foulkes WD. BMC Med Genomics. 2011;4:75.</small>

1. **RNA sequencing reveals the role of splicing polymorphisms in regulating human gene expression**  
   <small>Lalonde E, **Ha KCH**, Wang Z, Bemmo A, Kleinman CL, Kwan T, Pastinen T, Majewski J. Genome Res. 2011;21:545–54.</small>

1. **Comparison of Affymetrix Gene Array with the Exon Array shows potential application for detection of transcript isoform variation**  
   <small>**Ha KCH**, Coulombe-Huntington J, Majewski J. BMC Genomics. 2009;10:519.</small>

1. **Germ-line DNA copy number variation frequencies in a large North American population**  
  <small>Zogopoulos G, **Ha KCH**, Naqib F, Moore S, Kim H, Montpetit A, Robidoux F, Laflamme P, Cotterchio M, Greenwood C, Scherer SW, Zanke B, Hudson TJ, Bader GD, Gallinger S. Hum Genet. 2007;122:345–353.</small>


